components/QuantitySelector.js

```diff
@@ -0,0 +1,87 @@
+import styled from 'styled-components';
+import useStore from '../hooks/useStore';
+
+export default function QuantitySelector({ product }) {
+  const addQuantity = useStore(state => state.addQuantity);
+  const subtractQuantity = useStore(state => state.subtractQuantity);
+  const removeQuantity = useStore(state => state.removeQuantity);
```

Comment:
```suggestion
  const setQuantity = useStore(state => state.setQuantity);
```

Only use setQuantity, referenced in the useStore review